llvm/test/MC/AsmParser/labels.s

```diff
@@ -30,7 +30,7 @@ foo:
         .long "a 0"
 
 // CHECK: .section "a 1,a 2"
-//.section "a 1", "a 2"
+.section "a 1", "a 2"
```

Comment:
```
<stdin>:24:2: note: possible intended match here
 .section a 1,"a 2"
```

Reply:
Hmmm that looks like incorrect syntax for the directive. I suggest you comment-out the .section directive and make it a COM: CHECK: